frameworks/Java/voovan/benchmark_config.json

```diff
@@ -1,7 +1,7 @@
 {
   "framework": "voovan",
   "tests": [{
-    "default": {
+    "mvc": {
```

Comment:
This needs to be called default otherwise the toolset gives off lots of warnings

Reply:
i fix this to "default", thanks